modules/apps/segments/segments-test/src/testIntegration/java/com/liferay/segments/web/internal/context/test/SegmentsDisplayContextTest.java

```diff
@@ -85,13 +119,161 @@ public void setUp() throws Exception {
 			null);
 
 		_serviceTracker.open();
+
+		_user = UserTestUtil.addUser();
+
+		_mockLiferayPortletRenderRequest = _renderPortlet();
 	}
 
 	@After
 	public void tearDown() {
 		_serviceTracker.close();
 	}
 
+	@Test
+	public void testGetActionDropdownItems() {
+		List<DropdownItem> dropdownItems = ReflectionTestUtil.invoke(
```

Comment:
Better if we create a new method _ getActionDropdownItems to hide the reflectivity.